tests/nlu/utils/test_hf_transformers.py

```diff
@@ -134,328 +138,236 @@ def test_attention_mask(
+            ["Good evening.", "here is the sentence I want embeddings for."],
+            [(3, 768), (9, 768)],
             [
-                "Good evening.",
-                "hello",
-                "you're",
-                "r. n. b.",
-                "rock & roll",
-                "here is the sentence I want embeddings for.",
+                [-0.3092685, 0.09567838],
+                [0.02152853, -0.08026707, -0.1080862, 0.12423468, -0.05378958],
             ],
             [
-                ["Good", "even", "ing"],
-                ["hello"],
-                ["you", "re"],
-                ["r", "n", "b"],
-                ["rock", "&", "roll"],
                 [
-                    "here",
-                    "is",
-                    "the",
-                    "sent",
-                    "ence",
-                    "I",
-                    "want",
-                    "embed",
-                    "d",
-                    "ings",
-                    "for",
+                    -0.03930358216166496,
+                    0.034788478165864944,
+                    0.12246038764715195,
+                    0.08401528000831604,
+                    0.7026961445808411,
                 ],
-            ],
-            [
-                [(0, 4), (5, 9), (9, 12)],
-                [(0, 5)],
-                [(0, 3), (4, 6)],
-                [(0, 1), (3, 4), (6, 7)],
-                [(0, 4), (5, 6), (7, 11)],
                 [
-                    (0, 4),
-                    (5, 7),
-                    (8, 11),
-                    (12, 16),
-                    (16, 20),
-                    (21, 22),
-                    (23, 27),
-                    (28, 33),
-                    (33, 34),
-                    (34, 38),
-                    (39, 42),
+                    -0.018586941063404083,
+                    -0.09835464507341385,
+                    0.03242188319563866,
+                    0.09366855770349503,
+                    0.4458026587963104,
                 ],
             ],
         ),
     ],
 )
-@pytest.mark.skip_on_windows
-def test_hf_transformer_edge_cases(
-    model_name, model_weights, texts, expected_tokens, expected_indices
+def test_hf_transformers_shape_values(
+    model_name,
+    model_weights,
+    texts,
+    expected_shape,
+    expected_sequence_vec,
+    expected_cls_vec,
 ):
+    if model_name == "bert" and not model_weights and bool(os.environ.get("CI")):
+        pytest.skip(
+            "Reason: this model is too large, loading it results in"
+            "crashing of GH action workers."
+        )
 
-    if model_weights is None:
-        model_weights_config = {}
-    else:
-        model_weights_config = {"model_weights": model_weights}
-    transformers_config = {**{"model_name": model_name}, **model_weights_config}
+    config = {"model_name": model_name, "cache_dir": HF_TEST_CACHE_DIR}
+
+    if model_weights:
+        config["model_weights"] = model_weights
```

Comment:
Same, could be separated out in a function.

Reply:
I separated this out, but since it's really only relevant for this and `LMfeaturizer` I didn't put it into conftest. I know this is unconventional, but I'd like to avoid confusion. Once `HFTransformer` is deprecated, the method will definitely belong just in `LMFeaturizer`

Reply:
Open for counter arguments, though :)

Reply:
Makes perfect sense 👍